README.md

```diff
@@ -15,10 +15,6 @@ A backup application for the [Android Open Source Project](https://source.androi
 
 For older versions of Android, check out [the branches](https://github.com/stevesoltys/seedvault/branches).
 
-## Getting Started
```

Comment:
Any reason for removing this? The wiki contains a lot of helpful information.

Reply:
I created #164 to track the issue of it being outdated.

Reply:
Well, I think that having basic AOSP integration in README is nice, wiki isn't up-to-date, and it's redundant to have both in the README. Also the section is called "Getting started", but to me "getting started" would explain how to build it ouf of tree and install it manually. But that's really up to you. I can remove the README.md changes for the moment, and just have the overlay.

Reply:
I guess we can do just the README changes in a separate PR.

Wiki helps - there's more there than just the default overlay, a patch is linked to actually enable this for existing users as well (who may have this set to the default), and not just new users/installs.